telemetry/packages/public-app/components/DisplacementChart.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+
+import { Card, LineChart, Title } from '@tremor/react';
+import { useQuery } from 'react-query';
+import format from 'date-fns/format';
+
+const valueFormatter = (number: any) =>
+  `$ ${new Intl.NumberFormat('uk').format(number).toString()}`;
+
+export const DisplacementChart = () => {
+  const { data, isLoading, isError } = useQuery(
+    'displacement',
+    async () =>
+      (await fetch(
+        `${process.env.NEXT_PUBLIC_TELEMETRY_SERVER}/pods/pod_1/public-data/displacement?start=0`,
```

Comment:
could convert these fetches into a custom function somewhere else to make it a bit cleaner